Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -227,3 +227,23 @@ public enum AuthMechanism: String, Decodable {
     case mongodbX509 = "MONGODB-X509"
     case plain = "PLAIN"
 }
+
+public let INVALID_ARGUMENT_ERROR = InvalidArgumentError(message: "")
+public let AUTHENTICATION_ERROR = AuthenticationError(message: "")
+public let LOGIC_ERROR = LogicError(message: "")
+
+extension CommandError {
```

Comment:
`CommandError` sometimes needs to be constructed from some non-`@testable` imported tests, so I added this constructor here to avoid the `@testable` import _just_ for error construction.